tests/integration/dab_rbac/test_crud_permissions.py

```diff
@@ -109,8 +110,9 @@ def test_add_permissions(
                     related_perm = "view"
                 give_obj_perm(default_user, related_obj, related_perm)
 
-    response = user_client.post(url, data=post_data, format="json")
-    assert response.status_code == 201, response.data
+    with override_settings(WEBHOOK_BASE_URL="https://www.example.com/"):
```

Comment:
@mkanoor Would it better if we set this as a decorator for the test function? Not sure how much difference it makes, but since this test is parameterized, this override gets run for every parameter.